### PR TITLE
Add getUrlVariables API to EdgeIdentity package

### PR DESCRIPTION
**Pull request by cacheung**
Add getUrlVariables API

## Description
Add getUrlVariables API,  update sample app, update doc.  

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/edgeidentity/android/src/main/java/com/adobe/marketing/mobile/reactnative/edgeidentity/RCTAEPEdgeIdentityModule.java
Comment (addb) [inline packages/edgeidentity/android/src/main/java/com/adobe/marketing/mobile/reactnative/edgeidentity/RCTAEPEdgeIdentityModule.java:81]: nit: indentation

File: packages/edgeidentity/package.json
Comment (cacheung) [reply packages/edgeidentity/package.json:3]: Adid's setAdvertisingIdentitifer is an API in core.   We had that when implementing direct identity.  I tested this API with latest  EdgeIdentity extension, consent adid works properly with this API.  I will add this API to the documentation.

File: packages/edgeidentity/ts/Identity.ts
Comment (emdobrin) [inline packages/edgeidentity/ts/Identity.ts:76]: Looks like the catch->reject case is not handled below

File: packages/edgeidentity/ios/src/RCTAEPEdgeIdentity.m
Comment (emdobrin) [inline packages/edgeidentity/ios/src/RCTAEPEdgeIdentity.m:60]: typo: getUrlVariables

File: apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.m
Comment (emdobrin) [inline apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.m:68]: Should AEPMobileOptimize be removed here?

File: packages/edgeidentity/README.md
Comment (emdobrin) [inline packages/edgeidentity/README.md:180]: nit: URL variables =

File: apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.m
Comment (cacheung) [reply apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.m:68]: The ci fails when I remove this, it is based in staging branch.  I have to leave  the AEPMobileOptimize here.  Planning  to merge this PR after Optimize PR, so should not be a problem.

Comment (emdobrin) [reply apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.m:68]: Just to confirm we are removing Optimize here and it will be added back with the Optimize PR?

**Review by emdobrin — APPROVED** [review]
Left a small question about the sample app dependencies, other than that looks good!

File: apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.m
Comment (cacheung) [reply apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.m:68]: I was able to remove the Optimize extension without CI failure.  Will keep it this way for edgeIdentity branch.  It will be added back once merge to staging with AJO extension PR changes.

**Pull request merged**
